### PR TITLE
Change various tournament output.sendReply's to output.errorReply's

### PR DESCRIPTION
**Pull request by Irraquated**
I've excluded the valid formats message and the valid types message as in my opinion they don't count as errors, but can be changed.

### CODE REVIEW

File: tournaments/index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function createTournamentGenerator(generator, args, output) {
 	var Generator = TournamentGenerators[toId(generator)];
 	if (!Generator) {
 		output.sendReply(generator + " is not a valid type.");
-		output.sendReply("Valid types: " + Object.keys(TournamentGenerators).join(", "));
+		output.errorReply("Valid types: " + Object.keys(TournamentGenerators).join(", "));
```
Comment (Zarel) [inline tournaments/index.js]: Why wouldn't you make the line above an errorReply as well?

Comment (Irraquated) [inline tournaments/index.js]: Damn, must've missed one. give me a moment.

**Comment by Irraquated** [comment]
Should be better now @Zarel

File: tournaments/index.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ function usersToNames(users) {
 function createTournamentGenerator(generator, args, output) {
 	var Generator = TournamentGenerators[toId(generator)];
 	if (!Generator) {
-		output.sendReply(generator + " is not a valid type.");
+		output.errorReply(generator + " is not a valid type.");
 		output.sendReply("Valid types: " + Object.keys(TournamentGenerators).join(", "));
```
Comment (ascriptmaster) [inline tournaments/index.js]: He said to make the above line an `errorReply` **as well**, which means this line should also be an `errorReply`.

**Comment by Zarel** [comment]
I would include "Valid types" and "Valid formats" in `errorReply` so they'll be the same color, which will make it more obvious to users that they're part of the same message.

**Comment by Irraquated** [comment]
Sigh... will be changing from Atom and going back to using a simple editor.

**Comment by Irraquated** [comment]
Ack it just gets worse and worse...

**Comment by Irraquated** [comment]
Couldn't imagine something so simple, to take this bad of a turn...

**Comment by Irraquated** [comment]
Will fix this in the morning, I guess i'm just too tired to fix this right now. :sleeping: :zzz:

File: tournaments/index.js [hunk 3]
```diff
@@ -1012,4 +1012,4 @@ exports.createTournament = createTournament;
 exports.deleteTournament = deleteTournament;
 exports.get = getTournament;
 
-exports.commands = commands;
+exports.commands = commands;
```
Comment (kotarou3) [inline tournaments/index.js]: Keep the trailing newline please

**Pull request closed without merging**